tasks/image-segmentation/task.yaml

```diff
@@ -20,30 +25,36 @@ param_groups:
     required: false
 - name: parameters
   params:
+  - name: validation
+    description: Whether to perform training or validation for image segmentation. If true, only validation
+      will be performed; if false, only training will be performed
+    value: true
```

Comment:
Training should be the default mode for this template, so this parameter should be `false`